### PR TITLE
Minor updates on PHP documentation.

### PR DESCRIPTION
**Pull request by GhaziTriki**
It should slightly improve the code quality.

### CODE REVIEW

**Comment by WouterSioen** [comment]
:+1:

File: src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php [hunk 1]
```diff
@@ -44,7 +44,8 @@ public function testInstallationProcess()
 
     /**
      * @param \Symfony\Component\DomCrawler\Crawler|null $crawler
-     * @param \Symfony\Bundle\FrameworkBundle\Client $client
+     * @param \Symfony\Bundle\FrameworkBundle\Client|null $client
```
Comment (WouterSioen) [inline src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php]: Why the `|null` part? (in both crawler and client?) We always expect a Crawler and a Client object. With null, our methods won't work.

Comment (WouterSioen) [inline src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php]: Also in the other methods in this class

Comment (WouterSioen) [inline src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php]: @GhaziTriki can you fix this?

**Comment by GhaziTriki** [comment]
Another commit is coming in next hour for more improvements.

**Comment by WouterSioen** [comment]
great!

**Comment by GhaziTriki** [comment]
It should be the last commit for this pull request. I have noticed that there a lot of PSR-2 incompatibilities after running a fix with php-cs-fixer. Are you happy for me to create open an issue to discuss about rules to consider?

**Comment by WouterSioen** [comment]
Yes, better PSR2 compliance is always welcome!

**Pull request merged**